Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -2751,7 +2905,7 @@ def main():
             raise
 
         demisto.error(traceback.format_exc())
-        return_error(str(err))
+        return_error(f'Error: {str(err)}\nTrace: \n{traceback.format_exc()}')
```

Comment:
revert, no need to print the trace in the war-room

Reply:
Why not? When you have an error in one of your methods it helps to know where.
It solved an issue for me during the development here

Reply:
the traceback will be in the server log, there is no reason to print it in the war-room. it's not very esthetic.